libs/scripts/scripts.js

```diff
@@ -25,6 +25,7 @@ const stageDomainsMap = {
   'www.stage.adobe.com': {
     'www.adobe.com': 'origin',
     'helpx.adobe.com': 'helpx.stage.adobe.com',
+    'business.adobe.com': { to: 'business.stage.adobe.com', removeExt: true },
```

Comment:
Shouldn't the extension always be removed on stage.adobe.com? Or are there special cases

Reply:
I'm not aware of any special cases, but I'm trying to keep the solution consumer agnostic in case there are

Reply:
Should we do the reverse in this case, so that this will work without changes to the configs?

`addHtmlExt` if we need it anywhere? Having the extension should be the special case

Reply:
For BACOM we have the .html extension but for BACOM Blog we do not.

- https://business.stage.adobe.com/resources/main.html ->
  - https://main--bacom--adobecom.hlx.page/resources/main
- https://business.stage.adobe.com/blog/tags/the-latest ->
  - https://main--bacom-blog--adobecom.hlx.page/blog/tags/the-latest
  
We use `htmlExclude` to prevent appending .html to the blog pages. 
https://github.com/adobecom/bacom/blob/stage/scripts/scripts.js#L124-L126

Perhaps the `useDotHtml` config and `appendHtmlToLink` would fit these use cases?

Reply:
@mokimo I initially considered whether this should be an opt-in or opt-out type of config setting. The current approach felt more agnostic, but for our consumers, I see the value of enabling this by default, with the option to opt-out through the config. I’ve made the change 👍.

Reply:
@Brandon32 Another enhancement will come to also consider paths when determining which conversion to make. That will fix the problems with use-cases like `business.state.adobe.com` <-> `business.state.adobe.com/blog `